node/src/transition_frontier/candidate/transition_frontier_candidate_actions.rs

```diff
@@ -52,131 +52,50 @@ pub enum TransitionFrontierCandidateAction {
         hash: StateHash,
         error: SnarkBlockVerifyError,
     },
-    DetectForkRange {
-        hash: StateHash,
-    },
-    ShortRangeForkResolve {
-        hash: StateHash,
-    },
-    LongRangeForkResolve {
-        hash: StateHash,
-    },
     #[action_event(level = info)]
-    BestTipUpdate {
-        hash: StateHash,
-    },
     TransitionFrontierSyncTargetUpdate,
```

Comment:
should `TransitionFrontierSyncTargetUpdate` contain level=info? seems accidental

Reply:
fixed